js/ui/paygUnlockDialog.js

```diff
@@ -373,6 +377,34 @@ var PaygUnlockDialog = GObject.registerClass({
         return buttonsBox;
     }
 
+    _createEntryArea() {
+        let head, tail = null;
```

Comment:
I don't think this does what you expect it to do – it assigns `null` to `b` and leaves `a` as `uninitialized`. I think these variables should just be declared where they are used.